frontend/components/employee/employee.js

```diff
@@ -0,0 +1,194 @@
+import {Grid} from "@mui/material";
+import * as React from "react";
+import {useState} from "react";
+import {CustomDatePicker, CustomAutoComplete, InputTextField, DropDownList} from '../form/formComponent';
+import Button from "@mui/material/Button";
+import DialogActions from "@mui/material/DialogActions";
+
+
+const EmployeeForm = (props) => {
+
+    const titleList = [
```

Comment:
Is it possible to store these const as ENUM in a separate file so that we may re-use them in some other places?

Reply:
This title list would be used only in the employee so we might not need an ENUM.